src/Mvc/src/Microsoft.AspNetCore.Mvc.Core/breakingchanges.netcore.json

```diff
@@ -72,5 +72,35 @@
     "TypeId": "public abstract class Microsoft.AspNetCore.Mvc.ModelBinding.ObjectModelValidator : Microsoft.AspNetCore.Mvc.ModelBinding.Validation.IObjectModelValidator",
     "MemberId": "public abstract Microsoft.AspNetCore.Mvc.ModelBinding.Validation.ValidationVisitor GetValidationVisitor(Microsoft.AspNetCore.Mvc.ActionContext actionContext, Microsoft.AspNetCore.Mvc.ModelBinding.Validation.IModelValidatorProvider validatorProvider, Microsoft.AspNetCore.Mvc.ModelBinding.Validation.ValidatorCache validatorCache, Microsoft.AspNetCore.Mvc.ModelBinding.IModelMetadataProvider metadataProvider, Microsoft.AspNetCore.Mvc.ModelBinding.Validation.ValidationStateDictionary validationState)",
     "Kind": "Addition"
+  },
+  {
+    "TypeId": "public class Microsoft.AspNetCore.Mvc.MvcOptions : System.Collections.Generic.IEnumerable<Microsoft.AspNetCore.Mvc.Infrastructure.ICompatibilitySwitch>",
+    "MemberId": "public System.Boolean get_AllowBindingHeaderValuesToNonStringModelTypes()",
+    "Kind": "Removal"
```

Comment:
We should consider leaving these implementations here, **or** coming up with a better design. I don't really relish us adding and removing this (and churning supressions) each release. 

Adding/removing properties is semantically meaningful, but this isn't

Reply:
This is about removing properties (specifically their getters and setters). We decided in an offline email thread to
> Completely remove the properties we no longer want to support i.e. those enabling most of the legacy behaviours.

Are you suggesting reopening that conversation?

Reply:
Maybe I'm confused. I thought you also un-implemented the interfaces.

Reply:
This is what I'm talking about: https://github.com/aspnet/AspNetCore/pull/4628/files#diff-d9489227febd545e09ee512b868b9b8dL16